pkgs/development/python-modules/mkdocs/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, python3
+, fetchFromGitHub
+, buildPythonPackage
+, fetchpatch
+, click
+, jinja2
+, markdown
+, mergedeep
+, pyyaml
+, pyyaml-env-tag
+, ghp-import
+, importlib-metadata
+, watchdog
+, packaging
+, Babel
+, mock
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs";
+  version = "1.2.3";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-LBw2ftGyeNvARQ8xiYUho8BiQh+aIEqROP51gKvNxEo=";
+  };
+
+  patches = [
+    # TODO: remove this patch on the next release
+    # the upstream patch https://github.com/mkdocs/mkdocs/commit/c93fc91e4dc0ef33e2ea418aaa32b0584a8d354a.patch
+    # should work, but doesn't because tox.ini doesn't apply cleanly
+    # and fetchpatch with excludes doesn't seem to have any effect
+    ./fix-tests-no-tox-ini.patch
```

Comment:
If we don't need the tox changes we can use `fetchpatch` with `exclude` instead of vendoring the commit into nixpkgs.

Reply:
I tried that, it didn't work. I wrote as much in the comment.

Reply:
Ah, I using this incorrectly and using the fixed-output from _before_ adding `excludes` 🤦🏻 . Glad this works!

Reply:
Yeah, that's a bit tricky. Silly me missed your comment regarding the excludes.